doc/processing.md

```diff
@@ -0,0 +1,92 @@
+      within the application.
+    - If no potential matches to any existing facility are found, the application
+      creates a new facility using the country, name, and address and assigns a new
+      OAR ID number to the new facility.
+
+Contributed rows are matched to the existing facility list using statistical
+text analysis. The application calculates the probability that a contributed row
+is a match to an existing facility by comparing the country, name, and address
+fields individually and then joining those three comparisons together to create
+a final confidence score. The three individual comparisons are not given equal
+weight, however. The country of a submitted row, for example, must be an exact
+match.
+
+The application has a set of examples apparel facility names and addresses that
+are used to "train" the matching process. There are examples of similar items
+that should not be considered matches as well as examples of dissimilar items
+that should be considered matches. Using this training data allows the
+application to mathematically determine which differences in the text of
+facility name or address are significant and which are not.
+
+There are a some categories of text differences that are filtered out before the
+statistical matching process takes place. These items _do not_ affect whether a
+contributed row is a match to an existing facility:
+
+- Capitalization
+- Extra whitespace characters between words
+- The presence or absence of the following punctuation characters - / , : '
+
+## Common Questions
+
+> Why are only some of the facilities from my list appearing in searches?
+
+Each row from a contributed CSV file is processed separately. Rows that are
+matched successfully or rows from which new OAR facilities are created will
+appear in searches immediately after processing. If the application requires the
+contributor to verify a potential match, the facility will not be searchable
+until the match is approved or rejected. If the application encounters an error
+while processing a row, it will not appear in search results.
+
+> One of the items in my list has an "ERROR" status. How can I tell what happened?
+
+When viewing the list, clicking on a row in an "ERROR" status will show
+additional information regarding why the processing failed.
+
+>How long will it take to process my contributed facility list?
+
+List items are scheduled for processing as soon as they are uploaded, but it is
+difficult to say with certainty how long it will take to complete the
+processing. The size of the list and the number of other lists being processed
+at the same time will affect how long it will take to complete the processing of
+all the items in a list. If all the items in a contributed list still have the
+initial status of "UPLOADED" after one hour, please send an email to
+info@openapparel.org with the contributor name and the name of the list.
```

Comment:
Would it be worth adding a question like "How are OAR IDs generated?" whose answer would document how that works?

Reply:
One more potential question:

```
> What happens when I confirm or reject a match?
```

Reply:
👍 Natalie and Katie liked these questions. I will add them.

Reply:
Added in 0a0426d